src/Digdir.Domain.Dialogporten.Infrastructure/InfrastructureExtensions.cs

```diff
@@ -81,6 +81,7 @@ public static class InfrastructureExtensions
         {
             Duration = TimeSpan.FromMinutes(20),
             // The resource list is several megabytes and might take a while to process
+            FactorySoftTimeout = TimeSpan.FromSeconds(5),
```

Comment:
Remove this for now and just implement the logging change? So can we hopefully work out what happens if/when this happens again.